outputs.tf

```diff
@@ -30,42 +30,41 @@ output "svpc_host_project_id" {
 }
 
 output "subnets_names" {
-  value       = google_compute_subnetwork.subnetwork.*.name
+  value       = [for network in google_compute_subnetwork.subnetwork : network.name]
```

Comment:
Why do you use a `for` here and in the other outputs, instead of the simpler splat expression `google_compute_subnetwork.subnetwork[*].name`? I find that `for` it more verbose and less readable (especially for people not yet familiar with 0.12 syntax).

Reply:
Agreed, I had to use `for` because using the spat syntax in combination changed the behavior of the outputs and broke them, it's also why I added tests for the outputs.  When using the `for_each`  some of the outputs weren't correctly being created.  There's a resolved conversation somewhere in this PR, let me see if I can find that direct link!

Reply:
@ludoo here's the conversation! https://github.com/terraform-google-modules/terraform-google-network/pull/73#discussion_r328489995